sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_intel_matrix.asciidoc

```diff
@@ -322,11 +322,126 @@ q.submit([&](sycl::handler& cgh) {
+    size_t stride, layout Layout);
+
+// Only available when Layout != layout::dynamic
+// and when std::is_same_v<T1, std::remove_const_t<T2>>
+template <typename Group, typename T1, typename T2,
+          size_t Rows, size_t Cols, use Use, layout Layout,
+          typename PropertyListT>
+void joint_matrix_load(Group g,
+    joint_matrix<Group, T1, Use, Rows, Cols, Layout> &res,
+    ext::oneapi::experimental::annotated_ptr<T2, PropertyListT> src,
+    size_t stride);
+
+} // namespace sycl::ext::intel::experimental::matrix
+```
+```c++
+namespace sycl::ext::intel::experimental::matrix {
+
+template <typename Group, typename T1, typename T2, size_t Rows, size_t Cols,
+          typename PropertyListT>
+void joint_matrix_store(Group g,
+   const joint_matrix<Group, T1, use::accumulator, Rows, Cols, layout::dynamic> &res,
+   ext::oneapi::experimental::annotated_ptr<T2, PropertyListT> dest,
+   size_t stride, layout Layout);
+
+// Only available when Layout != layout::dynamic
+template <typename Group, typename T1, typename T2,
+          size_t Rows, size_t Cols, use Use, layout Layout,
+          typename PropertyListT>
+void joint_matrix_store(Group g,
+    joint_matrix<Group, T1, Use, Rows, Cols, Layout> &res,
+    ext::oneapi::experimental::annotated_ptr<T2, PropertyListT> src,
+    size_t stride);
+
+} // namespace sycl::ext::intel::experimental::matrix
+```
+==== Example
+
+```c++
+using syclex = sycl::ext::oneapi::experimental;
+using syclintelex = sycl::ext::intel::experimental;
+
+auto aptr = annotated_ptr(ptr, syclex::properties{syclintelex::cache_control_read<cache_control_read_type::cached, cache_level::L2>});
+q.parallel_for(nd_range<2>(G, L), [=](nd_item<2> it) {
+  sub_group sg = it.get_sub_group();
+  joint_matrix<sub_group, bfloat16, use::a, tM, tK, layout::row_major> tA;
+  for (int k = 0; k < K; k += tileK) {
+    // User specifies that this load will be cached to L2
+    joint_matrix_load(sg, tA, aptr + sg_startx * tM * K + k, K);
+    ...
+  }
+});
+```
+
+=== Prefetch
+
+```c++
+namespace sycl::ext::intel::experimental::matrix {
+
+template <typename Group, typename T, typename PropertyListT, size_t Rows, size_t Cols,
+          typename Properties = ext::oneapi::experimental::empty_properties_t>
+void joint_matrix_prefetch(Group g, T* ptr, size_t stride, layout Layout,
+	  size_t Rows, size_t cols, Properties properties = {});
+
+} // namespace sycl::ext::intel::experimental::matrix
+```
+
+`joint_matrix_prefetch` allows groups of work-items to cooperatively
+prefetch `Rows x cols` data in a 2d manner. This function is a group
+function, as defined in Section 4.17.3 of the core SYCL
+specification.
+
+The level of cache targeted by `joint_matrix_prefetch` in the last
+argument is specified using the compile-time properties defined in the
+SYCL extension
+link:../../proposed/sycl_ext_oneapi_prefetch.asciidoc[sycl_ext_oneapi_prefetch.asciidoc#specifying-cache-levels]
```

Comment:
This link doesn't work in the HTML render.  I think the "#specifying-cache-levels" part needs to be outside of the square brackets.